plugins/flow-visibility/policy-recommendation/policy_recommendation_job.py

```diff
@@ -0,0 +1,586 @@
+import antrea_crd
+import datetime
+import json
+import kubernetes.client
+import random
+import string
+import uuid
+
+from policy_recommendation_utils import *
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import udf
+from pyspark.sql.types import StringType
+
+# Column names of flow record table in Clickhouse database used in recommendation job 
+FLOW_TABLE_COLUMNS = [
+    'sourcePodNamespace',
+    'sourcePodLabels',
+    'destinationIP',
+    'destinationPodNamespace',
+    'destinationPodLabels',
+    'destinationServicePortName',
+    'destinationTransportPort',
+    'protocolIdentifier',
+]
+
+NAMESPACE_ALLOW_LIST = [
+    'kube-system',
+    'flow-aggregator',
+    'flow-visibility'
+]
+
+ROW_DELIMITER = "#"
+PEER_DELIMITER = "|"
+
+def get_flow_type(destinationServicePortName, destinationPodLabels):
+    if destinationServicePortName != "":
+        return "pod_to_svc"
+    elif destinationPodLabels != "":
+        return "pod_to_pod"
```

Comment:
Do we expect to always store podLabels in the database even if the optional podLabels are set as false?

Reply:
Currently yes becasue we could not create policies without pod lables. 
We may need additional service inside k8s cluster to fetch Pod labels and store in dababase, if there is no podLabels fields when flow aggregator exported.